pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -119,7 +119,7 @@ type ClusterLogging struct {
 }
 
 type VisualizationStatus struct {
-	KibanaStatus []KibanaStatus `json:"kibanaStatus,omitempty"`
+	KibanaStatus []elasticsearch.KibanaStatus `json:"kibanaStatus,omitempty"`
```

Comment:
if the console can scrape this no problem then we should eventually do the same for the `elasticsearch.ElasticsearchStatus`

Reply:
I'd say everything related to ES should be in the ES operator. Because now we have `ElasticsearchStatus` in the CLO and `elasticsearch.ElasticsearchStatus` in the EO. I think we have to keep the second one. If more information is needed, the local `ElasticsearchStatus` should contain the EO version of the `ElasticseachStatus` in it, together with other fields.